Update this patch manually when doing the next major verbump after 3.5.x.

Gentoo bug #408025.

diff --git a/awesomeConfig.cmake b/awesomeConfig.cmake
index 009be17..932ea30 100644
--- a/awesomeConfig.cmake
+++ b/awesomeConfig.cmake
@@ -56,7 +56,7 @@ a_find_program(GZIP_EXECUTABLE gzip FALSE)
 a_find_program(LUA_EXECUTABLE lua FALSE)
 a_find_program(LDOC_EXECUTABLE ldoc.lua FALSE)
 # theme graphics
-a_find_program(CONVERT_EXECUTABLE convert TRUE)
+a_find_program(CONVERT_EXECUTABLE bin/convert TRUE)
 # doxygen
 include(FindDoxygen)
 # pkg-config
